### PR TITLE
(qmlpref 2, last round) remove showPin

### PR DESCRIPTION
**Pull request by janiversen**
### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [ ] New feature
- [ x] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Pull request long description:

continued cleanup of qmlprefs.

### CODE REVIEW

File: mobile-widgets/qml/CloudCredentials.qml [hunk 1]
```diff
@@ -46,31 +46,31 @@ Item {
 
 		Controls.Label {
 			text: qsTr("Email")
-			visible: !prefs.showPin
+			visible: PrefCloudStorage.cloud_verification_status !== CloudStatus.CS_NEED_TO_VERIFY
```
Comment (dirkhh) [inline mobile-widgets/qml/CloudCredentials.qml:49]: this isn't wrong.
but is it better?

File: mobile-widgets/qmlmanager.cpp [hunk 2]
```diff
@@ -339,10 +339,6 @@ void QMLManager::openLocalThenRemote(QString url)
 		appendTextToLog(QStringLiteral("%1 dives loaded from cache").arg(dive_table.nr));
 		setNotificationText(tr("%1 dives loaded from local dive data file").arg(dive_table.nr));
 	}
-	if (qPrefCloudStorage::cloud_verification_status() == qPrefCloudStorage::CS_NEED_TO_VERIFY) {
-		appendTextToLog(QStringLiteral("have cloud credentials, but still needs PIN"));
```
Comment (dirkhh) [inline mobile-widgets/qmlmanager.cpp:343]: so we are missing this fairly useful piece of information in the log

**Review by dirkhh — COMMENTED** [review]
Yes, this removes some code.
I am not convinced that this is an improvement.
`showPin` is very easy to understand.
I'll go along with it, but this seems fairly arbitrary to me.
I sometimes like to have a variable somewhere that describes a state that the app is in.

**Comment by janiversen** [comment]
I would agree with your statement, if showPin was a cover functions for something like
return (PrefCloudStorage.cloud_verification_status === CloudStatus.CS_NEED_TO_VERIFY), but it is a separate variable, meaning it is difficult to know if there are situations where:
showPin == true and
PrefCloudStorage.cloud_verification_status !== CloudStatus.CS_NEED_TO_VERIFY)

Of course I could have chosen to make such a cover function, but I preferred a clean replace, which is easy to review, and then maybe later (once the oldStatus issue is solved) make something more elegant.

File: mobile-widgets/qmlmanager.cpp (hunk 2 above)
Comment (janiversen) [reply mobile-widgets/qmlmanager.cpp:343]: it's back.

**Comment by dirkhh** [comment]
good explanation. Makes sense.

**Comment by dirkhh** [comment]
merged

**Pull request closed without merging**